test/connectFactoryObservable.test.tsx

```diff
@@ -20,9 +27,43 @@ describe("connectFactoryObservable", () => {
   })
   describe("hook", () => {
     it("returns the latest emitted value", async () => {
-      const [useNumber] = connectFactoryObservable((id: number) => of(id))
+      const valueStream = new BehaviorSubject(1)
```

Comment:
You could simplify this by removing the ID from the output. It is not necessary for this test.

Reply:
Good catch!